src/Views/SimulatorView.css

```diff
@@ -3,10 +3,10 @@
 
   display: grid;
   grid-template-columns: 1fr 1fr;
-  grid-template-rows: 30vw 1fr;
+  grid-template-rows: 60vh 1fr;
```

Comment:
This property defines the height of each row, so I thought it was a little strange we were using the view width unit `vw` to define it. Not a biggie either way, happy to revert this.

Reply:
this was to ensure the simulator had a consistent aspect ratio as the page resized.